File: app/src/main/java/mega/privacy/android/app/receivers/CameraEventReceiver.java
package mega.privacy.android.app.receivers;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;

import static mega.privacy.android.app.utils.JobUtil.*;


public class CameraEventReceiver extends BroadcastReceiver {
    
    @Override
    public void onReceive(final Context context,Intent intent) {
        startCameraUploadServiceIgnoreAttr(context);
    }
}

File: app/src/main/java/mega/privacy/android/app/receivers/CameraEventReceiver.kt
package mega.privacy.android.app.receivers

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent
import mega.privacy.android.app.utils.JobUtil
import timber.log.Timber

class CameraEventReceiver : BroadcastReceiver() {
    override fun onReceive(context: Context, intent: Intent) {
        Timber.d("CameraEventReceiver")
        JobUtil.fireCameraUploadJob(context)
    }
}
